README.md

```diff
@@ -51,29 +51,28 @@ Here’s a complete `package.json` example, for a hypothetical module named `@mo
   "type": "module",
   "main": "./dist/index.js",
   "exports": {
-    ".": "./src/moment.mjs",
-    "./": "./src/util/",
-    "./timezones/": "./data/timezones/",
-    "./timezones/utc": "./data/timezones/utc/index.mjs"
+    "/": "./src/util/",
+    "/timezones/": "./data/timezones/",
+    "/timezones/utc": "./data/timezones/utc/index.mjs"
   }
 }
 ```
 
-Within the `"exports"` object, the key string after the `'.'` is concatenated on the end of the name field, e.g. `import utc from '@momentjs/moment/timezones/utc'` is formed from `'@momentjs/moment'` + `'/timezones/utc'`. Note that this is string manipulation, not a file path: `"./timezones/utc"` is allowed, but just `"timezones/utc"` is not. The `.` is a placeholder representing the package name. The main entrypoint is therefore the dot string, `".": "./src/moment.mjs"`.
+Within the `"exports"` object, the key string from the `'/'` is concatenated on the end of the name field, e.g. `import utc from '@momentjs/moment/timezones/utc'` is formed from `'@momentjs/moment'` + `'/timezones/utc'`. Note that this is string manipulation, not a file path: `"./timezones/utc"` is allowed, but just `"timezones/utc"` is not. The `.` is a placeholder representing the package name. The main entrypoint is therefore the dot string, `".": "./src/moment.mjs"`.
```

Comment:
> Note that this is string ...

Seems like the last few sentences need updating.  `"./timezones/utc"` -> `"/timezones/utc"` and "The `.` is a placeholder" -> "The `/` is a placeholder".